dev/com.ibm.ws.app.manager/src/com/ibm/ws/app/manager/internal/ApplicationConfig.java

```diff
@@ -200,6 +200,20 @@ public boolean getUseJandex() {
         return _applicationManager.getUseJandex();
     }
 
+    public String getAnnotationScanLibrary() {
```

Comment:
Should this logic move to the `EARDeployedAppInfo` class and be similar to how we handle `webModuleClassPathLoader`?  Similarly the `ANNOTATION_SCAN_LIBRARY` constants comes out of ApplicationManagerConstants and moves to `EarDeployedAppInfo`